test/mixins.py

```diff
@@ -245,6 +245,14 @@ def bigflow_spawn(self, cmd, **kwargs):
         return self.subprocess_spawn(cmd, **kwargs)
 
 
+class IteratorAssertionsMixin(Mixin):
+    def assertEmpty(self, iterator: Iterator):
```

Comment:
It is better to use `Iterable` (well, it is only a typehint, so doesn't matter too much).
BTW any iterator should be an `Iterable` in Python.

Also add parameter `msg` to the function with default value `"Iterable should be empty"`, so user may
redefine that message.